pages/curriculum/[lesson].tsx

```diff
@@ -1,25 +1,53 @@
 import * as React from 'react'
+import { useQuery } from '@apollo/react-hooks'
+import { useRouter } from 'next/router'
 import Layout from '../../components/Layout'
 import LessonTitleCard from '../../components/LessonTitleCard'
 import Alert from '../../components/Alert'
 import ChallengeMaterial from '../../components/ChallengeMaterial'
+import { GET_LESSONS } from '../../graphql/queries'
+import { Challenge } from '../../@types/challenge'
+
+type Lesson = {
+  id: number
+  title: string
+  description: string
+  order: number
+  challenges: Challenge[]
+  docUrl: string
+}
```

Comment:
Doesn't curriculum page also have a Lesson type already? Could we update that one and not have to redifine a new type?